src/Intercom/Clients/UsersClient.cs

```diff
@@ -399,19 +399,30 @@ public User RemoveCompanyFromUser(User user, List<String> companyIds)
             }
 
             ClientResponse<User> result = null;
-            String body = JsonConvert.SerializeObject(new { id = user.id, companies = companyIds.Select(c => new { id = c, remove = true })});
+            String body = JsonConvert.SerializeObject(new { id = user.id, companies = companyIds.Select(c => new { id = c, remove = true }) });
             result = Post<User>(body);
-            return result.Result;   
+            return result.Result;
         }
 
-        private String Transform (User user)
+        private String Transform(User user)
         {
             List<Company> companies = null;
 
-            if (user.companies != null && user.companies.Any ())
-                companies = user.companies.Select (c => new Company () { id = c.id, company_id = c.company_id }).ToList ();
+            if (user.companies != null && user.companies.Any())
+            {
+                companies = user.companies.Select(c => new Company()
+                {
+                    remote_created_at = c.remote_created_at,
```

Comment:
Curious, how exactly does this code work? What values are there going to be on the both sides of the = ?